website_event_questions_by_ticket/__manifest__.py

```diff
@@ -4,7 +4,7 @@
 {
     "name": "Conditional Events Questions",
     "summary": "Events Questions conditional to the chosen ticket",
-    "version": "14.0.1.0.0",
+    "version": "15.0.1.0.0",
```

Comment:
Your migration commit message should be:

[MIG] website_event_questions_by_ticket: Migration to 15.0